### DISCUSSION TITLE
Dietpi-Software | Flaresolverr

### DISCUSSION THREAD

**Discussion by umbertix** [MichaIng/DietPi#6176] [Software Request]
#### Formal software information
- Software title | flaresolverr
- Short description | A proxy to bypass Cloudflare and DDos guard on some indexers
- GitHub URL (if available) | https://github.com/FlareSolverr/FlareSolverr
- (Official) install documentation | https://github.com/FlareSolverr/FlareSolverr#installation

#### Are there similar/alternative software titles available with DietPi-Software?
- No

#### What makes your requested software better than the above solutions, if available?
- N/A

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
- Docker image is available
- Python package installation is also available

#### Can you provide the installation steps that you would suggest DietPi-Software to do?
1. Sorry, I am totally inexperienced in bash coding

#### Are you willing to help maintaining the software installation, e.g. in case of needed setup changes due to updates etc.? This is not needed, but could speed up our decision to implement it, as man power is always a topic :wink:.
- Sorry, I am totally inexperienced in bash coding

**Comment by Joulinar** [comment]
Many thanks for your request.

I converted it into a discussion, so people can vote for it via ⬆️ button at above post.
